src/main/kotlin/uk/gov/justice/digital/hmpps/esurveillanceapi/resource/TestResource.kt

```diff
@@ -24,12 +24,12 @@ class TestResource(
 
   @PostMapping("/sms")
   fun sendSms(@RequestBody request: SmsRequest): ResponseEntity<String> {
-    val notificationId = notifyService.sendSms(
```

Comment:
I think we can get rid of both /sms & /email endpoints and related code (SmsRequest/EmailRequest)